src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js

```diff
@@ -880,6 +882,33 @@ var adyenHelperObj = {
 
     return JSON.parse(resultObject.getText());
   },
+
+  createGiftCardPM(parsedGiftCardObj, divideBy, order) {
+    let paymentInstrument;
```

Comment:
Shall we create a new file for this? AdyenHelper is already huge and it's better we limit it to simple utility or helper functions specific to Adyen.